webview/src/plots/components/templatePlots/templatePlotsSlice.ts

```diff
@@ -57,12 +57,12 @@ export const templatePlotsSlice = createSlice({
       return {
         ...state,
         hasData: !!action.payload,
+        nbItemsPerRow: action.payload.nbItemsPerRow,
         plotsSnapshots: snapShots,
         sections:
           JSON.stringify(plotSections) === JSON.stringify(state.sections)
             ? state.sections
-            : plotSections,
-        size: Math.abs(action.payload.size)
```

Comment:
`Math.abs` was there because when first implementing the feature, I kept making the width negative. It shouldn't be a problem now.